libs/expo/shared/apollo/tsconfig.json

```diff
@@ -2,12 +2,14 @@
   "extends": "../../../../tsconfig.base.json",
   "compilerOptions": {
     "module": "commonjs",
+    "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "strict": true,
     "noImplicitOverride": true,
     "noPropertyAccessFromIndexSignature": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "jsx": "react"
```

Comment:
We now have `tsx` in the lib so it appears that `esModuleInterop` and `jsx` is now necessary.